hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/RatisPipelineUtils.java

```diff
@@ -99,24 +99,6 @@ static void destroyPipeline(DatanodeDetails dn, PipelineID pipelineID,
     }
   }
 
-  /**
-   * Return the list of pipelines who share the same set of datanodes
-   * with the input pipeline.
-   *
-   * @param stateManager PipelineStateManager
-   * @param pipeline input pipeline
-   * @return list of matched pipeline
-   */
-  static List<Pipeline> checkPipelineContainSameDatanodes(
-      PipelineStateManager stateManager, Pipeline pipeline) {
```

Comment:
Is this not required?

Reply:
yes, it seems not used. for now , we use the following one instead
```
  static List<Pipeline> checkPipelineContainSameDatanodes(
      StateManager stateManager, Pipeline pipeline) {
```